src/org/opendatakit/briefcase/ui/reused/source/ShowSourceForm.java

```diff
@@ -66,6 +95,22 @@ public JComponent getContainer() {
   public void setEnabled(boolean enabled) {
     super.setEnabled(enabled);
     resetButton.setEnabled(enabled);
+    reloadButton.setEnabled(enabled);
+  }
+
+  private void markReloadTimerElapsed() {
+    this.reloadTimerElapsed = true;
```

Comment:
I know he’s on holiday but we’ve gotta come back and remove these extra `this.`s.

Reply:
Okey, dokey. I’ll take care of them